node/packages/sdk-schema/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@serverless/sdk-schema",
   "repository": "serverless/console",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
Why not `v1.0.0` ?

Using `v0.0.1` prevents us from relying on semver entirely (each release will have to be accompanied with explicit bumps in all dependent packages)

Reply:
Good catch, I wasn't thinking of that.

Reply:
If you feel that schema is not stable (and that we will possibly make some breaking changes to it at the beginning), we may start with v0.1, and then shift to v1 once we will feel that all properties are used and not to be altered

Reply:
I say we do `v0.1.0` for now now that I'm thinking about it some more. Even though I feel it is pretty stable what we have defined, better to be extra safe while we are still setting up all the initial schemas.